app/admin/top-issues/components/TopIssueDisplay.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import { SVGIconChooser } from '@shared/buttons/SVGIconChooser';
+import TextField from '@shared/inputs/TextField';
+import { ButtonGroup } from '@mui/material';
+import Button from '@mui/material/Button';
+import { FaCheck, FaEdit } from 'react-icons/fa';
+import { FaXmark } from 'react-icons/fa6';
+import Image from 'next/image';
+import IconButton from '@mui/material/IconButton';
+import { updateTopIssue } from './TopIssuesList';
+
+export const TopIssueDisplay = ({
+  issue,
+  topIssues = [],
+  setTopIssues = () => {
+  },
+}) => {
+  const [editTopIssueId, setEditTopIssueId] = useState(null);
+  const [editTopIssueName, setEditTopIssueName] = useState(null);
+  const [editTopIssueIcon, setEditTopIssueIcon] = useState(null);
+
+  const handleIssueEdit = issue => () => {
```

Comment:
```  
const handleIssueEdit = (issue) => {
```

Reply:
This will change the signature of the thunk, changing it to a plain function?

How will the closure for the `issue`, out of the iterated list work then?

This would be a breaking change for the currently standing changes, right? 🤔